templates/data_flow/show.html.twig

```diff
@@ -6,74 +6,82 @@
 {% block header %}{{ 'DataFlow'|trans }}{% endblock %}
 
 {% block toolbar %}
-    <a href="{{ path('data_flow_index') }}">{{ 'Back to data flow list'|trans }}</a>
+    <a class="btn btn-outline-secondary btn-sm" href="{{ path('data_flow_index') }}">{{ 'Back to data flow list'|trans }}</a>
 {% endblock %}
 
 {% block content %}
-    {% embed '_table.html.twig' %}
-        {% block tbody %}
-            <tr>
-                <th>{{ 'Name'|trans }}</th>
-                <td>{{ data_flow.name }}</td>
-            </tr>
-            <tr>
-                <th>{{ 'Description'|trans }}</th>
-                <td>{{ data_flow.description }}</td>
-            </tr>
+
+    <p class="text-muted small mb-1">{{ 'Flow name'|trans }}</p>
+    <h2 class="h5">{{ data_flow.name }}</h2>
+
+    {% if data_flow.description %}
+        <div class="col-12 bg-white shadow-sm rounded-lg">
+            <p class="text-muted small mb-1">{{ 'Description'|trans }}</p>
+            <p class="lead">{{ data_flow.description }}</p>
+        </div>
+    {% endif %}
+    {% embed '_table.html.twig' with {'hover': 'null'} %}
```

Comment:
I suppose you mean (the value) `null` as opposed to (the string) `'null`' here. And then you could just use `{% embed '_table.html.twig' %}` and `hover|default('table-hover')` in the embedded template will handle the missing value correctly.

If you're trying to tell the embedded template to *not use any hover* when passing `{hover: null}`, then you need to use something like `{{ hover is defined and hover is null ? '' : hover|default('table-hover') }}`.

Also, you don't have to write `{'hover': …}` – `{hover: …}` will suffice.

Reply:
Thanks for clarifying. Wasn't sure how to correctly approach this.